src/wp-content/themes/twentytwentythree/styles/marigold.json

```diff
@@ -1,305 +1,317 @@
 {
   "$schema": "https://schemas.wp.org/trunk/theme.json",
-  "title": "Marigold",
+  "version": 2,
+	"title": "Marigold",
```

Comment:
Looks like the indentation is off here. I can fix this pre-commit, and we can sync the change back to WordPress/twentytwentythree after.

Reply:
Thank you for picking this up!